index.js

```diff
@@ -0,0 +1,80 @@
+const hamburguerMenuIcon = document.querySelector('.three-bar-menu');
+// const crossIcon = document.querySelector('.crossIcon');
```

Comment:
- [ ] The mobile menu (X) button, does not disappear when clicked. I can see that the DOM selector of this button is commented out. Please ensure to improve upon this aspect